packages/sakuli-rollup-hooks/src/ts-support/enable-typescript-command.class.ts

```diff
@@ -0,0 +1,81 @@
+import {CommandModuleProvider, CommandModule, Argv} from "@sakuli/core";
+import execa from 'execa';
+import { createInterface } from "readline";
+import {stripIndents} from 'common-tags'
+import { resolve, isAbsolute, join } from "path";
+import {promises as fs} from 'fs';
+import chalk from 'chalk'
+import ora, { Ora } from 'ora';
+import { defaultTsConfig } from "./default-ts-config.const";
+import { getInstalledPresets } from "./get-installed-presets.function";
+import packageTypingMapping from "./package-typing-mapping.const";
+import { EOL } from "os";
+
+const userInput = async (question: string): Promise<string> => {
+    const rl = createInterface(process.stdin,process.stdout);
+    return new Promise<string>((res, rej) => {
+        rl.question(question, answer => {
+            res(answer);
+            rl.close();
+        });
+    })
+}
+
+export const enableTypescriptCommand: CommandModuleProvider = (): CommandModule => {
+    return ({
+        command: 'enable-typescript [project]',
+        describe: `Enables Typescript support for the provided project`,
+        builder(argv: Argv): any {
+            return argv.positional('project', {
+                describe: 'Path to the project, can be absolute or relative to $PWD'
```

Comment:
Maybe we should clarify that with `project` the actual npm project where Sakuli is installed is meant, not a particular test suite etc.

Reply:
I think the term `project` is used in the same way as we use it in the docs on sakuli.io.

But we have to be careful for upcoming features like #22 to use the same wording.